Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -219,6 +219,12 @@ const ScrollView = React.createClass({
      */
     pagingEnabled: PropTypes.bool,
     /**
+     * When true, ScrollView allows use of pinch gestures to zoom in and out.
+     * The default value is false.
```

Comment:
Currently, pinch gesture is always enabled by default and there is no way to disable it.